src/classes/HH_Container_LCTRL.cls

```diff
@@ -518,26 +518,16 @@ public with sharing class HH_Container_LCTRL {
     public static SObject getHHNamesGreetings(SObject hh, list<Contact> listCon) {
         try {
             // update naming exclusions field based on exclusion checkboxes
-            for (Contact con : listCon) {
-                HH_Households_TDTM.copyNamingExclusionsFromCheckboxes(con);
-            }
+            Contacts contacts = new Contacts(listCon);
```

Comment:
Instantiations of this domain class as just `contacts` will be easily confused will collections of Contacts. I know you would prefer to keep the class name as is, but either way I think instances of the class have to be named differently - such a `contactsDomain` or `contactsService` or `domainContacts`, etc. Just something to differentiate from a `List<Contact>`.

Reply:
For two main reasons I am still hesitant to do this.  
1. As mentioned in another comment, all instances of `List<Contacts>` should change over time to be instances of `Contacts` - there will also be fewer usages over time as behavior is moved to be encapsulated inside the Contacts domain class
2. This kind of naming is bakes the Type into the variable name.  Apex is strongly typed, so putting the Type in the name is a bit redundant, and, while we are seeing a lot of general var names of 'contacts' here, the local variable name can always be named to be more expressive for the variables usage, i.e., `Contacts contactsForNewsletter = new Contacts(List<Contact>)`